packages/notionable/src/base-trade/sagas/vault-risk.ts

```diff
@@ -97,14 +103,40 @@ export function postVaultAccountRisk(
             ?.interestRate
         );
 
+        const vaultCapacity =
+        network && vaultAddress
+          ? Registry.getConfigurationRegistry().getVaultCapacity(
+              network,
+              vaultAddress
+            )
+          : undefined;
+
+        let underMinAccountBorrow = false;
```

Comment:
Should we put this value into the trade state and then read it back out on the component? No sense in doing the calculation twice...

Reply:
I think our code would be a lot cleaner if we put all the logic for `useVaultCapacity` and `useVaultActionErrors` inside the observable `vault-manager`, maybe we can pull these changes out and put them in a separate PR to handle that.